lib/ClipCopy/tests/ClipCopy-test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { expect } from 'chai';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+
+import { mount, setupApplication } from '../../../tests/helpers';
+import CalloutHarness from '../../../tests/CalloutHarness';
+import ClipCopyInteractor from './interactor';
+import ClipCopy from '../ClipCopy';
+
+const ClipCopyMock = props => (
+  <CalloutHarness>
+    <ClipCopy {...props} />
+  </CalloutHarness>
+);
+
+//
+// this should work, but it doesn't, and I give up.
+// I can run
+//    expect(cc.hasClipboard).to.be.true;
```

Comment:
I am wondering whether the trick for waiting for the `hasClipboard` existence would help similar to how we use occasionally timer for stuff to mount. 

The idea is to add the following in the interactor and then wait for it at the end of `beforeEach`. It may though run into the same issue.
```
  whenLoaded() {
    return this.when(() => this. hasClipboard);
  }
```

Reply:
Alas:
```
  beforeEach(async () => {
    await setupApplication();
    await mount(<ClipCopyMock text="some text" />);
    await cc.whenLoaded();
  });
```
still results in 
```
FAILED TESTS:
  ClipCopy
    ✖ "before each" hook for "shows a clipboard icon"
      Chrome 84.0.4147 (Mac OS X 10.14.6)
    Error: convergent assertion returned `false`
        at loop (tests/index.js:41569:17)
```